src/transformers/models/bert_generation/modeling_bert_generation.py

```diff
@@ -41,6 +43,478 @@
 _TOKENIZER_FOR_DOC = "BertGenerationTokenizer"
 
 
+# Copied from transformers.models.bert.modeling_bert.BertEmbeddings
+class BertEmbeddings(nn.Module):
```

Comment:
```suggestion
class BertGenerationEmbeddings(nn.Module):
```
All prefixes should be adapted to the new model (so here and all other classes below up to `BertEncoder`->`BertGenerationEncoder`. Some of your copied from statements will then need to be completed (I'll give on example below).